### PR TITLE
Fix NPE during property to locator conversion

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java [hunk 1]
```diff
@@ -22,16 +22,16 @@
 
 public class StringToLocatorConverter implements Converter<String, Locator>
 {
-    private LocatorConversionUtils conversionUtils;
+    private final LocatorConversionUtils conversionUtils;
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java:25]: I might be wrong but this may lead to circular dependencies

**Review by valfirst — APPROVED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java (hunk 1 above)
Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java:25]: Sure thing it could, but the build looks ok? Won't usage of setter hide circular dependency if we have any?

Comment (uarlouski) [reply vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java:25]: - previously it also looked good before that failed first time, I mean that this thing does not reproduce stably
- didn't get your second question :)

**Review by uarlouski — APPROVED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java (hunk 1 above)
Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/java/org/vividus/spring/StringToLocatorConverter.java:25]: Let me rephrase :)
We should not workaround circular deps in most of the cases, they should be broken. So I would like to be aware if we got one here :)
